site2/docs/concepts-messaging.md

```diff
@@ -150,6 +150,8 @@ In *Key_Shared* mode, multiple consumers can attach to the same subscription. Me
 
 ![Key_Shared subscriptions](assets/pulsar-key-shared-subscriptions.png)
 
+**Key_Shared subscription is a beta feature. You can disable it at broker.config.**
```

Comment:
@codelipenghui shouldn't this be "You can enable it"?

Reply:
@sijie Key_shared subscription is enabled by default.  so i think we need to tell users can disable it at broker.config

Reply:
There was a comments above to enable it by default.

Reply:
Ah I see. sorry I missed that.